.gitignore

```diff
@@ -7,3 +7,4 @@ test/output/*
 tmp
 TODO.md
 .DS_Store
+.idea
```

Comment:
I use WebStorm (IntelliJ) and my IDE likes to dump all sorts of files into a project. This prevents them from being checked into Git.